Kickstarter-iOS/Views/Controllers/PledgeAmountViewController.swift

```diff
@@ -40,16 +36,14 @@ final class PledgeAmountViewController: UIViewController {
       |> ksr_addSubviewToParent()
       |> ksr_constrainViewToEdgesInParent()
 
-    _ = ([self.titleLabel, self.adaptableStackView], self.rootStackView)
+    _ = ([self.titleLabel, self.adaptableStackView, UIView(frame: .zero)], self.rootStackView)
```

Comment:
This makes sure that the vertical stack view will always align it's subviews to top (vertically)